content/kb/troubleshooting-websocket_handshake_error_with_unexpected_response_code_200.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Troubleshooting: Longhorn-UI: Error during WebSocket handshake: Unexpected response code: 200 #2265"
+author: Clark Hsu
+draft: false
+date: 2021-03-09
+catelogies:
+  - "longhorn-ui"
+---
+
+## Applicable versions
+
+Exiting Longhorn versions < v1.1.0 upgrade to Longhorn >= v1.1.0.
+
+## Symptoms
+
+After upgraded Longhorn to version >= v1.1.0, and encounter the following:
+
+Ingress messages:
+
+```
+{"level":"error","msg":"vulcand/oxy/forward/websocket: Error dialing \"10.17.1.246\": websocket: bad handshake with resp: 200 200 OK","time":"2021-03-09T08:29:03Z"}
+```
+
```

Comment:
There should be similar logs in longhorn manager pods. We can mention it as well